guides/v2.2/cloud/release-notes/cloud-tools.md

```diff
@@ -33,7 +33,12 @@ The following updates describe the latest improvements to the `{{site.data.var.c
 
     -  <!-- MAGECLOUD- 2799 -->Added a [Cron container]({{page.baseurl}}/cloud/docker/docker-development.html#cron-container) based on the PHP-CLI image.
 
--  <!-- MAGECLOUD- 2575 -->**Configure with PHP constants**—Added support for PHP constants in the `.magento.env.yaml` configuration file.
+-  **Environment variable updates**
+
+    -  <!-- MAGECLOUD- 2575 -->**Configure with PHP constants**—Added support for PHP constants in the `.magento.env.yaml` configuration file.
+
+    -  <!--MAGECLOUD-2879-->Added the [ENABLE_GOOGLE_ANALYTICS]({{page.baseurl}}/cloud/env/variables-deploy.html##enable_google_analytics) deploy environment variable to enable or disable Google Analytics on Staging and Integration environments.
```

Comment:
The word deploy is in a strange place. We don't describe them as deploy environment variables. We usually say environment variable for the deploy stage...or something similar.